lib/wallet/cardano-wallet.cabal

```diff
@@ -173,7 +174,7 @@ library
     , statistics
     , stm
     , streaming-commons
-    , strict-containers
+    , cardano-strict-containers
```

Comment:
Finally! 💯 

This means we can now depend upon the hackage version of `strict-containers` (if we need to).